specs/light_client/sync_protocol.md

```diff
@@ -177,16 +101,15 @@ The verification procedure is as follows:
 
 ```python
 def verify_block_validity_proof(proof: BlockValidityProof, validator_memory: ValidatorMemory) -> bool:
-    assert proof.shard_parent_block.beacon_chain_root == hash_tree_root(proof.header)
-    committee = compute_committee(proof.header, validator_memory)
+    assert proof.shard_parent_block.core.beacon_chain_root == hash_tree_root(proof.beacon_block_header)
+    committee = compute_persistent_committee_at_slot(validator_memory, compute_epoch_of_shard_slot(shard_parent_block.slot))
     # Verify that we have >=50% support
-    support_balance = sum([v.effective_balance for i, v in enumerate(committee) if proof.shard_bits[i]])
-    total_balance = sum([v.effective_balance for i, v in enumerate(committee)])
+    support_balance = sum([balance for i, (pubkey, balance) in enumerate(committee) if proof.shard_aggregation_bits[i]])
```

Comment:
Why 50%?

Reply:
To maximize fault tolerance against both safety and liveness attacks :stuck_out_tongue: